### PR TITLE
Issue-40: always load filter from URL state

### LINKED ISSUE #40: https://kanban.ist/board  URL does not always clear filters
Hi there,

I have created a bunch of filtered "views" and saved them as bookmarks (allows me to get there quickly without defining different views each time). I also created a bookmark to https://kanban.ist/board with the intent of "clearing filters" without the multiple clicks to go in to do so. But when I do that - the filters remain.

I suspect it is the way the clear filters button itself works with filtering that is not getting called when using the URL in the address bar....

Ideas? Thank you.

### PR DESCRIPTION
**Pull request by mishawakerman**
Resolves #40 

Previously we were loading the filter state from localStorage if we didn't have any filters saved in the URL. This would load the previous filter. We now *always* use the URL as the source of the filter.

tl;dr I've learn a lot since I first wrote this persistence logic :)

### CODE REVIEW

**Comment by mwakerman** [comment]
Ergh, multiple accounts!
